CHANGELOG.md

```diff
@@ -8,17 +8,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Migration Notes
 
-#### CUMULUS-3536 Upgrading from Aurora Serverless V1 to V2
```

Comment:
Hmmmmm 🤔 🤔 

18.5.0 is a backport. Do we want these entries to only appear in the 18.5.0 release? I don't think so because I think we want the next forward release (v19.1.0) to also have these entries because it is based on main, not this 18.x release series?

Does that make sense? I'm not 100% confident. I'm asserting we want the entries for these tickets in both releases.

Reply:
Wow this is confusing. After a Slack thread and too much time thinking about it, I suggest we not remove these entries on main. The 19.1.0 release will have them (e.g. 3536), even though they already exist on v19. 

Very open to argument though as this is weird 😆

Reply:
@npauzenga discussion happening in Slack but one idea is to just update 18.5.0 entry to call out this is a backport and may or may not contain duplicate changes. We've done that before (e.g. 18.3.4):

>**Please note** changes in v18.3.4 may not yet be released in future versions, as this
is a backport/patch release on the v18.3.x series of releases.  Updates that are
included in the future will have a corresponding CHANGELOG entry in future releases.